pyasic/web/vnish.py

```diff
@@ -58,7 +58,7 @@ async def send_command(
         allow_warning: bool = True,
         privileged: bool = False,
         **parameters: Any,
-    ) -> dict:
+    ) -> dict | None:
```

Comment:
None is returned in case no calls to API were success (for example - in case of 404 error)